### PR TITLE
Added expansion and invalid OP_CODES

### PR DESCRIPTION
**Pull request by Jouzo**
#### What kind of PR is this?:
/kind feature

#### What this PR does / why we need it:

Adds expansion and invalid opcodes mentioned in #107 

#### Which issue(s) does this PR fixes?:

Fixes missing opcodes in #107

### CODE REVIEW

**Comment by jingyi2811** [comment]
Nice!

**Review by jingyi2811 — DISMISSED** [review]

**Review by fuxingloh — CHANGES_REQUESTED** [review]
minor whitespace to remove

**Comment by Jouzo** [comment]
Where about ? It is not caught be the linter

File: packages/jellyfish-transaction/src/script/expansion.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  constructor () {
+    super(0xb1, 'OP_CHECKLOCKTIMEVERIFY')
+  }
+}
+
+/**
+ * This instruction allows a transaction output to be made unspendable until some point in the future.
+ * @See OP_CHECKLOCKTIMEVERIFY
+ */
+
```
Comment (fuxingloh) [inline packages/jellyfish-transaction/src/script/expansion.ts:26]:
```suggestion
```

here

**Review by fuxingloh — CHANGES_REQUESTED** [review]
my bad, I didn't attach it within my review

File: packages/jellyfish-transaction/src/script/expansion.ts (hunk 1 above)
Comment (Jouzo) [reply packages/jellyfish-transaction/src/script/expansion.ts]: Fixed !

**Review by fuxingloh — APPROVED** [review]

**Review by jingyi2811 — APPROVED** [review]

**Pull request merged**